pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2250,8 +2252,13 @@ func handleGlobalConfigImpl(ctxArg interface{}, key string,
 			updateVgaAccess(ctx)
 			ctx.setInitialVgaAccess = true
 		}
-		if gcp.GlobalValueInt(types.MetricInterval) != 0 {
-			ctx.metricInterval = gcp.GlobalValueInt(types.MetricInterval)
+		metricInterval := gcp.GlobalValueInt(types.MetricInterval)
+		if metricInterval != 0 && ctx.metricInterval != metricInterval {
```

Comment:
would you mind elaborating what is this condition for.

Reply:
I put small comment here